examples/tokens/token_create_transaction_kyc_key.py

```
@@ -450,50 +459,50 @@ def main():
         print("STEP 4: Creating a new account for KYC testing")
         print("=" * 70)
-        test_account_2, test_account_key_2 = create_account(
-            client, operator_key)
+        test_account_2, test_account_key_2 = create_account(client, operator_key)
         associate_token_to_account(
-            client, token_with_kyc, test_account_2, test_account_key_2)
+            client, token_with_kyc, test_account_2, test_account_key_2
+        )
 
         # Try to transfer without KYC (may fail)
         transfer_without_kyc_result = attempt_transfer_without_kyc(
```

Ruff (0.14.8)

Local variable `transfer_without_kyc_result` is assigned to but never used

Remove assignment to unused variable `transfer_without_kyc_result`

(F841)